packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -952,6 +952,42 @@ const defaultConfigs = {
       },
     },
   },
+  ETHUSD: {
+    type: "medianizer",
+    minTimeBetweenUpdates: 60,
+    medianizedFeeds: [
+        { type: "cryptowatch", exchange: "coinbase-pro", pair: "ethusd" },
+        { type: "cryptowatch", exchange: "binance", pair: "ethusdt" },
+        { type: "cryptowatch", exchange: "kraken", pair: "ethusd" },
+    ],
+  },
+  iFARMUSD: {
+    type: "expression",
+    expression: `
+        FARMUSD = FARMETH_UNISWAP * ETHUSD;
```

Comment:
nit: would you be able to avoid defining the `ETHUSD` feed by doing the following:
```js
FARMUSD = FARMETH_UNISWAP / USDETH
```
?

That is a little less readable, so maybe you'd prefer to do the following:
```js
ETHUSD = 1 / USDETH;
FARMUSD = FARMETH_UNISWAP * ETHUSD;
```

Or is there some other reason you'd prefer to define the `ETHUSD` differently from the way we define the `USDETH`?

Reply:
TBH it was just copied from the XSUSHIUSD config. I will update it this way.